parachain/primitives/beacon/src/types.rs

```diff
@@ -179,32 +179,33 @@ impl<const COMMITTEE_SIZE: usize> SyncCommittee<COMMITTEE_SIZE> {
 
 /// Prepared G1 public key of sync committee as it is stored in the runtime storage.
 #[derive(Clone, PartialEq, Eq, Encode, Decode, TypeInfo, MaxEncodedLen)]
-pub struct SyncCommitteePrepared<const COMMITTEE_SIZE: usize> {
+#[scale_info(skip_type_params(MaxCommitteeSize))]
+pub struct SyncCommitteePrepared<MaxCommitteeSize: Get<u32>> {
 	pub root: H256,
-	pub pubkeys: [PublicKeyPrepared; COMMITTEE_SIZE],
+	pub pubkeys: BoundedVec<PublicKeyPrepared, MaxCommitteeSize>,
```

Comment:
Can you try this as an experiment and see if the errors go away?

```suggestion
	pub pubkeys: Box<[PublicKeyPrepared; COMMITTEE_SIZE]>,
```

Will need to import [sp_std::boxed::Box](https://paritytech.github.io/substrate/master/sp_std/boxed/struct.Box.html)

Reply:
@vgeddes Cool, it does work!